ui-react/src/components/accounts/Account.tsx

```diff
@@ -8,35 +8,49 @@ import StatusIndicator from './StatusIndicator';
 export type ActionButton = {
   icon: string;
   tooltipText: string;
-  clickHandler: () => void
-}
+  clickHandler: () => void;
+};
 
 type AccountProps = {
   account: AccountType;
-  actionButtons?: ActionButton[]
-}
+  actionButtons?: ActionButton[];
+};
 
-export default function Account({
-  account, actionButtons
-}: AccountProps) {
+export default function Account({ account, actionButtons }: AccountProps) {
   const containerStyles = [styles.container];
 
-  const badgeNumberLog = account.logs.find((log) => log.originalEvent && log.originalEvent.exportedTransactionsNum > 0);
+  const badgeNumberLog = account.logs.find(
+    (log) => log.originalEvent && log.originalEvent.exportedTransactionsNum > 0,
+  );
   if (!account.active) containerStyles.push(styles.notActive);
+  if (!actionButtons) containerStyles.push(styles.pointer);
```

Comment:
This is the change.

Although I'm not sute why you need this if, can you explain?